subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyGradleModuleMetadataSmokeTest.groovy

```diff
@@ -34,8 +34,7 @@ class ThirdPartyGradleModuleMetadataSmokeTest extends AbstractSmokeTest {
         given:
         BuildResult result
         useSample("gmm-example")
-        // TODO Test Kotlin 1.4
-        def kotlinVersion = TestedVersions.kotlin.latestStartsWith("1.3")
+        def kotlinVersion = TestedVersions.kotlin.latest()
```

Comment:
:o: With Kotlin 1.5 around the corner and https://github.com/gradle/gradle/pull/16852 already in flight it might be better to stick to 1.4 in this test for now
```suggestion
        def kotlinVersion = TestedVersions.kotlin.latestStartsWith("1.4")
```